### PR TITLE
B_001 Tests are passing again. Solved nested exception in CalculatorManager.java

### PR DESCRIPTION
**Pull request by Sephirot92**
Big thanks to @IzabelaRojek for spotting that.

None of the tests was passing (being ignored). I started to seach for possible solution.

Reason for it was that there was a nasted exception in CalculatorService.java - With Arek we have had createn dependency injection without constructor to pass it. To secure it from eventual problems i added qualifier. 

Tests are passing again.

### CODE REVIEW

File: src/main/java/com/superdevs/HealthOMeter/calculator/WHRCalculator.java [hunk 1]
```diff
@@ -9,9 +9,9 @@
 import java.math.RoundingMode;
 
 @Component
-public class WhrCalculator {
+public class WHRCalculator {
 
-    Logger logger = LoggerFactory.getLogger(WhrCalculator.class);
+    Logger logger = LoggerFactory.getLogger(WHRCalculator.class);
 
     public BigDecimal calculateWhr (BigDecimal waistRatio, BigDecimal hipRatio){
```
Comment (ganzes) [inline src/main/java/com/superdevs/HealthOMeter/calculator/WHRCalculator.java]: calculateWhr > calculateWHR
(camleCase > WaistHipsRatio, imo when named calculateWhr it's not camelCase correct way of naming classes, methods etc.)

File: src/test/java/com/superdevs/HealthOMeter/HealthOMeterApplicationTests.java [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 	@BeforeAll
 	static void loadTestData(){
-		context = new AnnotationConfigApplicationContext(CalculatorManager.class, CalculatorService.class, WhrCalculator.class);
+		context = new AnnotationConfigApplicationContext(CalculatorManager.class, CalculatorService.class, WHRCalculator.class);
 		calculatorService = context.getBean(CalculatorService.class);
 	}
```
Comment (ganzes) [inline src/test/java/com/superdevs/HealthOMeter/HealthOMeterApplicationTests.java:27]: This class should only be used for loadContext 
 e.g.

```
@SpringBootTest
class HealthOMeterApplicationTests.java{

    @Test
    void contextLoads() {
    }
}
```

if you want to make test for specific classes, method etc. then create new package and classes that will check your code.

**Review by ganzes — COMMENTED** [review]
There are some minor things to change, especially regarding tests - we should have specific tests classes, and definitely not put everything into one class.

File: src/test/java/com/superdevs/HealthOMeter/HealthOMeterApplicationTests.java (hunk 2 above)
Comment (Sephirot92) [reply src/test/java/com/superdevs/HealthOMeter/HealthOMeterApplicationTests.java:27]: Disagree. I mean by default you are right, but in future we will make refactor and split it.

**Review by ganzes — APPROVED** [review]

**Pull request merged**